settings.php

```diff
@@ -52,8 +59,8 @@
  * $config_directories['staging'] = '/home/myusername/config/staging';
  * @endcode
  */
-$config_directories['active'] = 'files/config_' . md5($database) . '/active';
-$config_directories['staging'] = 'files/config_' . md5($database) . '/staging';
+$config_directories['active'] = 'files/config_' . md5(implode(',', $database)) . '/active';
+$config_directories['staging'] = 'files/config_' . md5(implode(',', $database)) . '/staging';
```

Comment:
Same as before, `serialize()` is probably a more robust way to convert an array to a string:

```suggestion
$config_directories['active'] = 'files/config_' . md5(serialize($database)) . '/active';
$config_directories['staging'] = 'files/config_' . md5(serialize($database)) . '/staging';
```